packages/rocketchat-ui-message/client/popup/messagePopup.js

```diff
@@ -192,6 +192,7 @@ Template.messagePopup.onCreated(function() {
 			return;
 		}
 		const value = template.input.value;
+		const preColon = value.substr(0, value.indexOf(':'));
```

Comment:
I have to this b/c when we select one emoji the string of the emoji replace the string on input-message. In this case, we need to keep the `+` caracter before the first `:`